messages/DefaultRuleManager.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
 	"warning": {
 		"targetSkipped": "Target: '%s' was not processed by any engines.",
-		"targetsSkipped": "Targets: '%s' were not processed by any engines."
+		"targetsSkipped": "Targets: '%s' were not processed by any engines.",
+		"pathDoubleProcessed": "A file was processed by both ESLint and ESLint-LWC, which may result in duplicate violations. Check out our documentation on target patterns to customize which paths each engine should target.",
+		"pathsDoubleProcessed": "Multiple files were processed by both ESLint and ESLint-LWC, which may result in duplicate violations. Check out our documentation on target patterns to customize which paths each engine should target."
```

Comment:
Would we be able to use a link shortener and do something like: "A file was processed by both ESLint and ESLint-LWC, which may result in duplicate violations. Check out the solution here: short url to documentation"

Reply:
@gkjung Hey, really nice job! 
A couple small things:
1) we try and avoid model verbs (may, should) as much as possible so we can do a small tweak for those
2) whenever possible, it's great to start with the action: "To customize which paths ..."
3) as much as it'd be awesome to include direct links to documentation, we don't, in case the doc links change over time. We don't want users experiencing broken links, you know?

Reply:
My slight rewrite suggesttions:

A file was processed by both ESLint and ESLint-LWC, which could result in duplicate violations. To customize which paths each engine targets, check out our documentation on target patterns.

Multiple files were processed by both ESLint and ESLint-LWC, which could result in duplicate violations. To customize which paths each engine targets, check out our documentation on target patterns.

Reply:
Clarifying question: a file was processed by two engines, and that could result in duplicates. But then the next sentence talks about paths, not files. So is it really that a node on the path was processed by two engines, and each path (and its nodes) be only checked by one engine? Just wondering how to connect file (first sentence) to path (second sentence) to make it most clear.

Reply:
That's a good point, I can swap out paths for files and it still keeps the meaning. The patterns allow them to specify what files are ran on a specific engine and if they have a pattern like "**/lwc/**" then any file whose path contains /lwc/ would get ran by that engine. 

I can change it to: "A file was processed by both ESLint and ESLint-LWC simultaneously, which could result in duplicate violations. To customize which files each engine targets, check out our documentation on target patterns."

Does that sound good? I realize I added "simultaneously". Josh suggested it and I thought I already added it.

Reply:
@gkjung Great, thanks for the explanation, and I agree, if file and path are synonymously, choose one for clarity. Looks good!

Reply:
Can we say, "Customize targetPatterns properties for eslint and eslint-lwc engines in ~/.sfdx-scanner/Config.json to remove overlap"?
We don't have an obvious section on target patterns that a user can easily look up.
Also, this message would need to take config file location as a parameter since the name/path differs between pilot and GA as well as across OS types.

Reply:
I'm good with that! Would we do this instead of adding the documentation in the other PR?

Reply:
How about we do both? We could add examples in the other PR to show what an overlap is and how they can work around it.